api/src/main/java/io/strimzi/api/kafka/model/JvmOptions.java

```diff
@@ -58,6 +59,16 @@ public void setXms(String xms) {
         this.xms = xms;
     }
 
+    @JsonProperty("-Xloggc")
+    @Description("-Xloggc option to to the JVM")
```

Comment:
Could we explain a bit more what it does?

Reply:
Will make it more descriptive (was copying the format of the other JsonProperty's directly above)

Reply:
Yeah, I think the descriptions for the others are bad as well - but we have to start somewhere to make it better ;-).